src/main/webapp/script.js

```diff
@@ -152,6 +150,32 @@ function createOmdbItemCard(omdbItem) {
   return card;
 }
 
+/**
+ * Makes the item submission button visible if the item is not a duplicate.
+ *
+ * @param { jQuery } submitButton - button used to submit omdb items
+ * @param { jQuery } itemCard - card div that displays info about the item found
+ * @param { JSON } omdbItem - the item found by omdb API
+ */
+function enableItemSubmissionIfUnique(submitButton, itemCard, omdbItem) {
+  fetch('/item-submission?imdbID=' + omdbItem.imdbID)
+      .then((response) => response.json())
+      .then((isItemUnique) => {
+        if (isItemUnique) {
+          submitButton.removeClass('d-none');
+          submitButton.off().one('click', () => {
+            submitItem(omdbItem);
+          });
+        } else {
+          itemCard.append($(
```

Comment:
A nice feature here would be to link to that item's page since the user probably wants to go to it. You can add a TODO here.

Reply:
Yeah I added a TODO there, probably going to add that feature in a future PR.